### PR TITLE
Update add vuln bio

### PR DESCRIPTION
**Pull request by Rick-Methot-NOAA**
## Concisely describe what has been changed/addressed in the pull request.
something went wrong in the merging of the add_vuln_bio issue.  Fixing it here.
#499 
* Resolves issue #
#499

### CODE REVIEW

File: SS_write_ssnew.tpl [hunk 1]
```diff
@@ -1487,15 +1487,15 @@ FUNCTION void write_nucontrol()
...
+    NuStart << F_reporting_ages << " # min and max age over which mean F will be calculated, with F=Z-M" << endl;
   }
   else
   {
     NuStart << "#COND 10 15 #_min and max age over which mean F will be calculated with F_reporting=4 or 5" << endl;
   }
   NuStart << F_std_basis_rd << " # F_std_scaling: 0=no scaling; 1=F/Fspr; 2=F/Fmsy; 3=F/Fbtgt; where F means annual F_std, Fmsy means F_std@msy; values >=11 invoke N multiyr (up to 9!) using 10's digit; >100 invokes log(ratio)" << endl;
   NuStart << double(mcmc_output_detail) + MCMC_bump << " # MCMC output detail: integer part (0=default; 1=adds obj func components; 2= +write_report_for_each_mceval); and decimal part (added to SR_LN(R0) on first call to mcmc)" << endl;
-  NuStart << ALK_tolerance << " # ALK tolerance ***disabled in code (example 0.0001)" << endl;
+  NuStart << ALK_tolerance << " # ALK tolerance ***disabled in code" << endl;
```
Comment (e-gugliotti-NOAA) [inline SS_write_ssnew.tpl:1498]: @Rick-Methot-NOAA so ALK tolerance is a disabled feature? If so I just need to update the documentation [here](https://github.com/nmfs-stock-synthesis/doc/blob/d02f2d4b50023e87c53baae729d115fd62ffdc86/6starter.tex#L242).

**Review by e-gugliotti-NOAA — DISMISSED** [review]
I see changes in the report and new files

File: SS_write_ssnew.tpl (hunk 1 above)
Comment (Rick-Methot-NOAA) [reply SS_write_ssnew.tpl:1498]: Yes, it is disabled.  Please update the documentation.

**Comment by Rick-Methot-NOAA** [comment]
candidate r4ss issues are columns in time_series report or catch report.   Also, the added table of selmax that occurs after the age_selectivity report.

**Review by iantaylor-NOAA — APPROVED** [review]
Looks good.
The error with testing r4ss was related to the new comment line which needed to be adjusted for. I've just done that in a branch and will merge once it passes the r4ss tests. At that point we can re-run the github action to confirm the fix for SS3.

**Comment by iantaylor-NOAA** [comment]
One more request, related to the already-merged previous pull request #499. 
Could you replace the blank line and add an underscore to change as follows?

From:
```

maximum ASEL2
```
to:
```
#
maximum_ASEL2
```

This makes it find the new output in r4ss and makes it consistent with other output tables (where blank lines are followed by sections calls "report:NN" and sub-sections within a table are separated by #).

**Comment by Rick-Methot-NOAA** [comment]
sorry about that Ian.  Thanks for reminding me of the agreed protocol.

**Comment by iantaylor-NOAA** [comment]
Thank you. I made one more minor change in 325dbd4. Hopefully that's OK too.

**Review by iantaylor-NOAA — APPROVED** [review]

**Comment by Rick-Methot-NOAA** [comment]
Thanks Ian

**Pull request merged**